langchain-core/langchain.config.js

```diff
@@ -66,6 +66,7 @@ export const config = {
     "utils/testing": "utils/testing/index",
     "utils/tiktoken": "utils/tiktoken",
     "utils/types": "utils/types/index",
+    "utils/is_openai_tool": "utils/is_openai_tool",
```

Comment:
Can we put this in an existing entrypoint? `types` maybe?

Reply:
Want to avoid fragmentation

Reply:
Moved to `@langchain/core/language_models/base` so it's exported from the same place as `ToolDefinition`